site/docs/latest

```diff
@@ -1 +1 @@
-3.0.3
\ No newline at end of file
+3.1.2
```

Comment:
IIUC, this would fix the latest doc issue. cc @HyukjinKwon

Reply:
Oh, is this a bug introduced at Today's @cloud-fan 's commit?

Reply:
Thank you for fixing this.

Reply:
yes

Reply:
I wrote a script to do this, and seems changing the latest link should be done manually as it's not always needed. I'll update my script.